### PR TITLE
uricontent: add test for DetectUriSigTest02

### PR DESCRIPTION
**Pull request by TheKharleeci**
Previous PR: #619

### CODE REVIEW

File: tests/uricontent-01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# *** Add configuration here ***
```
Comment (inashivb) [inline tests/uricontent-01/test.yaml:1]: nit: Please remove this line and the newline following this,

File: tests/uricontent-01/test.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    match:
+      event_type: flow
+- filter:
+    count: 1
+    match:
+      event_type: stats
+- filter:
+    count: 1
+    match:
+      event_type: alert
```
Comment (inashivb) [inline tests/uricontent-01/test.yaml:18]: Please add the signature ID which should alert among the three rules provided.

File: tests/uricontent-01/test.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+checks:
+- filter:
+    count: 1
+    match:
+      event_type: flow
+- filter:
+    count: 1
+    match:
+      event_type: stats
```
Comment (inashivb) [inline tests/uricontent-01/test.yaml:14]: We probably don't need the `stats` event but it doesn't hurt without the checks for other params, I think.

**Review by inashivb — CHANGES_REQUESTED** [review]
All in all, LGTM. :)
Some nits are requested from me.
Next steps:
1. Please check if any of the other tests in `detect-uricontent.c` do something differently. If yes, add tests corresponding to them too.
2. Make a folder for all these uricontent tests in the `tests` directory here.
3. Once all the convertible C tests have been converted to s-v, delete the corresponding tests from the C file and create a PR on https://github.com/OISF/suricata for that.

That should complete the process for this one file.

**Comment by TheKharleeci** [comment]
Followed By: #624

**Pull request closed without merging**